staging/src/k8s.io/apiextensions-apiserver/Godeps/Godeps.json

```diff
@@ -572,17 +572,14 @@
 		},
 		{
 			"ImportPath": "github.com/prometheus/client_golang/prometheus",
-			"Comment": "v0.9.2",
```

Comment:
What's happening here?  why is this stripping these out?

Reply:
I ran "./hack/update-staging-godeps.sh" and this just got dropped automatically.

Reply:
I did a rebase and after that "./hack/update-staging-godeps.sh" no longer generate additional godeps files.  So this was because my branch was out of sync with the master.  It is resolved now.